Code/Tools/AssetProcessor/Platform/Linux/AssetProcessor_Traits_Linux.h

```diff
@@ -7,5 +7,13 @@
  */
 #pragma once
 
+#include <climits>
+
+#ifndef PATH_MAX
+// this shouldn't happen
```

Comment:
Instead of a comment, this could be a descriptive compile time message
```suggestion
#pragma message "PATH_MAX not system defined. Setting to 4096"
```

Reply:
set the max length to 4096 as a fallback and added a compile time error message